src/addresses/Address.container.tsx

```diff
@@ -1,33 +1,26 @@
 import { FC, useContext, useEffect, useState } from 'react';
 import { Addresses } from './Address.component';
-import { AuthContext, K8sResourceCommon } from '../utils';
+import { AuthContext } from '../utils';
 import { useJolokiaServiceGetAddresses } from '../openapi/jolokia/queries';
 
-export type Address = {
-  name: string;
-};
-export interface AddressContainerProps {
-  brokerDetails: K8sResourceCommon;
-}
-
 const AddressContainer: FC = () => {
   const authToken = useContext(AuthContext);
 
   const { data: rawAddresses, isSuccess } = useJolokiaServiceGetAddresses({
     jolokiaSessionId: authToken,
   });
-  console.log('data', rawAddresses);
-  console.log('is success', isSuccess);
 
-  const [addressData, setAddressData] = useState<Address[]>([]);
+  const [addressData, setAddressData] = useState([]);
+
+  const getAddressData = (addresses: any[]) => {
+    return addresses.map((address: any) => ({
+      name: address.name,
+    }));
+  };
 
   useEffect(() => {
-    if (isSuccess) {
-      const AddressData = rawAddresses.map((address) => {
-        return {
-          name: address.name,
-        };
-      });
+    if (isSuccess && rawAddresses) {
+      const AddressData = getAddressData(rawAddresses);
```

Comment:
One last thing to fix, now that you've updated the yaml, there's no need to transform `rawAddresses` to something else. Just send it to the `addressData` parameter of the `Addresses` component.

Also it's an opportunity to rename `rawAddresses` to `addrresses` since it's not raw data that needs to be parsed anymore.

Reply:
okay sure, i will update.